src/background/services/accounts/handlers/avalanche_getAccountPubKey.ts

```diff
@@ -31,9 +33,6 @@ export class AvalancheGetAccountPubKeyHandler extends DAppRequestHandler {
   };
 
   handleUnauthenticated = ({ request }) => {
-    return {
-      ...request,
-      error: ethErrors.provider.unauthorized(),
-    };
+    return this.handleAuthenticated({ request });
```

Comment:
Do we want to open it up for the world? If we do this, any dapp can query the addresses of the wallet without user consent.